aicssegmentation/structure_wrapper/WorkflowEngine.py

```diff
@@ -33,6 +35,9 @@ def __init__(self, workflow_name: str, image: np.ndarray):
         self._data_folder: os.path = os.path.join(
             os.path.split(os.path.dirname(__file__))[0], "..", "demo_data"
         )
+        self.widget_info = None
+        with open(Path(__file__).parent.parent / "structure_wrapper_config" / f"all_functions.json") as file:
```

Comment:
[nit] I don't think `"all_functions.json"` needs to be an f-string?

Reply:
The linter was also complaining about this- removed